apps/masterbots.ai/components/routes/profile/user-card.tsx

```
@@ -140,33 +140,32 @@ export function UserCard({ user, loading }: UserCardProps) {
         setIsLoading(false)
       }
     }
-  }, [lastMessage, generateType, updateUserInfo])
+  }
 
   useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: handleUpdateUserInfo

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)